src/core/namespace.c

```diff
@@ -1754,6 +1762,20 @@ int setup_namespace(
                         };
                 }
 
+                /* Will be used to add bind mounts at runtime */
+                if (setup_propagate) {
+                        const char *root_dir_incoming = prefix_roota(root_directory, incoming_dir);
```

Comment:
this is problematic: mixing two kinds of stack allocation in the same expression

Reply:
uh it's an issue even for variable declaration? Had no idea - fixed. I wish the compiler would warn against these...